diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/ConsecutiveSpacesRuleFixTest.kt

```diff
@@ -0,0 +1,28 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.rules.ConsecutiveSpacesRule
+import org.cqfn.diktat.util.FixTestBase
+import org.junit.Test
```

Comment:
```suggestion
import org.junit.jupiter.api.Test
```
Your import is for junit 4, we are using junit 5. Because of the wrong import these tests are not picked up by runner, therefore your code coverage is lower than it should be. Please fix it.